official/vision/ops/augment.py

```diff
@@ -1869,7 +1869,7 @@ def _parse_policy_info(name: str,
   func = NAME_TO_FUNC[name]
 
   if level_std > 0:
-    level += tf.random.normal([], dtype=tf.float32)
+    level += level_std*tf.random.normal([], dtype=tf.float32)
```

Comment:
This will change the behavior completely. I am worrying about the effects of this. Have you done any tests verifying it won't break existing results using this augmentation ?

Reply:
It's OK to bring more flexibility, but the default behavior should keep backward compatibility.